W-11365480-FixGmailDoc-ESB (#286)
mulesoft/docs-composer: en/mulesoft/modules/ROOT/pages/ms_composer_gmail_reference.adoc

New email::

> This trigger fires on all emails (received, sent, or drafts), if no https://developers.google.com/gmail/api/guides/labels[filter] is specified.

Reviewer: ```suggestion
If you don't specify a  https://developers.google.com/gmail/api/guides/labels[filter^], this trigger fires on all emails (received, sent, or drafts).
```

Reviewer: This trigger fires on all emails (received, sent, or drafts) if you don't specify a  https://developers.google.com/gmail/api/guides/labels[filter^].
